### PR TITLE
90% - Post 7th Jul 2016 event 

### PR DESCRIPTION
**Pull request by scaleupcto**
Moving away from splashthat to a static site hosted on S3. 

This is the first version of that, it is a retrospective of the first event with a CTA for the mailing list.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,571 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (danielmatthew) [inline index.html:2]: 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+						<span class="img" style="background-image:url(./img/gallery1.jpg);">&nbsp;</span><span class="img" style="background-image:url(./img/gallery2.jpg);">&nbsp;</span>
+					</div>
+				</div>
+			</section>
+
+			<section class="event-catchup" id="catchup">
+				<div class="centered">
+					<div class="intro">
+						<h2>Follow the conversation</h2>
+						<span class="sub-heading">We saw 201 tweets during Codelicious #01. Find out what our delegates had to say on Twitter using <a href="https://twitter.com/search?q=codelicious">#Codelicious</a></span>
```
Comment (russellhill) [inline index.html]: maybe `target="_blank"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+				</div>
+				<div class="top_overlay"></div>
+			</section>
+
+			<section class="event-gallery" id="gallery">
+				<div class="gallery_overlay">
+					<div class="centered">
+						<h2>See all of the photos from Codelicious</h2>
+						<div class="all-gallery actions">
+							<a class="secondary-btn" href="https://www.flickr.com/photos/118197661@N02/albums/72157670869726615">View All Photos</a>
```
Comment (russellhill) [inline index.html]: maybe `target="_blank"`

File: index.html (hunk 1 above)
Comment (scaleupcto) [inline index.html:2]: Any more i11n please go at it - the hubspot template was pretty poor from what I saw!

**Comment by danielmatthew** [comment]
What's all the Wistia stuff used for?

**Comment by scaleupcto** [comment]
For the video highlights, we might just swap this out for a youtube link, Natalie is having a go on making something tonight.

**Pull request merged**